compiler/luci/import/src/Nodes/CircleTransposeConv.cpp

```diff
@@ -30,6 +30,13 @@ bool CircleTransposeConvGraphBuilder::validate(const ValidateArgs &args) const
   if (args.op.inputs.size() != 3)
     return false;
 
+  const auto &inputs = args.op.inputs;
+  const auto &tensors = args.reader.tensors();
+  const auto &filter_tensor = tensors.at(inputs[1]);
+  const auto &ifm_tensor = tensors.at(inputs[2]);
+  if (ifm_tensor.get()->shape.at(3) != filter_tensor.get()->shape.at(3))
```

Comment:
please use rank value not `3`

Reply:
@seanshpark 
Is it what I think it means?
```suggestion
  auto rank = ifm_tensor.get()->shape.size();
  if (ifm_tensor.get()->shape.at(rank - 1) != filter_tensor.get()->shape.at(rank - 1))
```

Reply:
> Is it what I think it means?

I think so. If this is the condition for TransposeConv. If we check with `3` then this code will break if rank is 2 or 3

Reply:
I don't know the detail but is `ifm_tensor.get()->shape.at(rank - 1) != filter_tensor.get()->shape.at(rank - 1)` must hold?

Reply:
Well, as [document](https://www.tensorflow.org/api_docs/python/tf/nn/conv2d_transpose#args) said, ifm and filter must be 4-D tensor. So, I think rank is always 4.

Reply:
OK then let's add rank check too if it's 4 for ifm and filter.